install_config/aggregate_logging.adoc

```diff
@@ -78,7 +88,7 @@ A set of files can be supplied in the secret:
 |*_server-tls.json_*
 |JSON TLS options to override the Kibana server defaults. Refer to
 https://nodejs.org/api/tls.html#tls_tls_connect_options_callback[Node.JS] docs
-for available options and the default options for an example.
```

Comment:
There's no good way for the user to see the default options without linking to the source code.